packages/functional-tests/tests/subscription-tests/coupons-ui-tests/coupon.spec.ts

```diff
@@ -1,5 +1,7 @@
 import { test, expect } from '../../../lib/fixtures/standard';
 
+test.describe.configure({ mode: 'parallel' });
```

Comment:
Based on the past few runs, I kinda think we should keep the `worker:1` setting. Tests have been completing in roughly 10-12mins, which is much faster than before. This might be due to them using less memory now and having to retry less often too.

Reply:
But if we stick w/ `workers:1` in the config, doesn't it mean this `{mode:'parallel'}` change is ignored?
Or am I maybe reading something wrong?

Reply:
I think so too, increasing workers might be good for CI. If for some reason it doesn't work out we can always change it back!

Reply:
Looks like with increasing workers to 2 the suite ran faster (execution time cut down to 10 mins from 12 mins ).